examples/utils/get-tinylicious-container/src/getTinyliciousContainer.ts

```diff
@@ -27,10 +27,12 @@ import { getContainer } from "./getContainer";
  */
 class InsecureTinyliciousUrlResolver implements IUrlResolver {
     public async resolve(request: IRequest): Promise<IResolvedUrl> {
-        const documentId = request.url.split("/")[0];
+        const parsedUrl = new URL(request.url);
```

Comment:
This looks like it probably shouldn't work here - InsecureTinyliciousUrlResolver doesn't really use real URLs, but instead the format `documentId/containerRelativePathing` (see the comment above).  And as a result `new URL(...)` won't be able to parse it correctly.

Reply:
Yeah, this won't work. This might be a good opportunity to add tests to verify this especially since we are using this in our hello world example. Unless there are tests somewhere that already test this package.

Reply:
sorry I didn't mean to push this change...i forgot to change it back